shub/plugins/pgp/models.py

```diff
@@ -180,13 +182,16 @@ class PGPPacketModel(models.Model):
     class Meta:
         abstract = True
 
-    def is_public_key(self):
+    @staticmethod
```

Comment:
Could you please test that these changes do not break the keystore?

We should probably discuss tests - if you want to more work on singularity registry it would be a very good use of time to get a test framework setup.

Reply:
Agreed.